### PR TITLE
KPI B updates

### PR DESCRIPTION
**Pull request by tomkralidis**
Discussions with @gaubert indicate that this KPI is ready for implementation. I've made some editorial updates as well as updated the parent document to include all KPIs accordingly.

### CODE REVIEW

File: KPI/B.adoc [hunk 1]
```diff
@@ -1,110 +1,85 @@
...
-* words are following Title case 
-* title is following WMCP guidance 
+=== What is being measured
 
-=== Why should we measure it?
-Title is the first reference to a resource displayed and is helps with initial identification. 
-If it's not available, too short, or too long then it makes it difficult for users understand the resource. 
+The title field shall always be populated and and follow the principles of
+WMCP guidance. The length shall not be too short or too long, contain less
+than 3 acronyms and represented in title case.
```
Comment (martaguti) [inline KPI/B.adoc:11]: Difficult to implement programmatically the acronyms check , unless you have an identified set of acronyms which may be difficult to build ?   We discussed to check whether all words are from an english dictionary (if that makes the check easier)

**Review by martaguti — COMMENTED** [review]
Nice simplification .. we discussed too how to stop people from adding (GTS bulleting headers TTAAii) into the titles , there a few wis records like that around the GISCS.
We could detect bulleting headers in the KPI implementation ( with regex...)

File: KPI/B.adoc (hunk 1 above)
Comment (tomkralidis) [reply KPI/B.adoc:11]: Thanks for the feedback @martaguti.  I've added a check that detects bulletin headers and applies a -1 score.

**Comment by tomkralidis** [comment]
Thanks for the feedback @martaguti. Regarding acronym and bulletin header checking, I think we have enough for now for an initial implementation (we can possibly have an acronym list and bulletin header regerx lookup as part of this KPI check in the tooling).

**Comment by tomkralidis** [comment]
@martaguti / @gaubert / @amilan17 updated per today's discussion for review/comment/approval.

File: KPI/B.adoc [hunk 2]
```diff
@@ -1,110 +1,75 @@
...
+for users to understand the resource.
+
+=== Rules for implementation
+
+.Good quality title implementation rules
+|===
+|Rule |Score
+
+|title exists
+|0
```
Comment (amilan17) [inline KPI/B.adoc:30]: change 0 to 1

File: KPI/B.adoc [hunk 3]
```diff
@@ -1,110 +1,75 @@
...
+|words in title are represented in "Title Case"
+|1
+
+|title contains less than 3 acronyms (words with all upper case)
+|1
+
+|title does not contain bulletin header (regular expression `[A-Z]{4}\d{2}[\s_]*[A-Z]{4}`)
+|1
+
+|abstract passes a basic spellcheck
```
Comment (amilan17) [inline KPI/B.adoc:50]: title, not abstract

File: KPI/B.adoc [hunk 4]
```diff
@@ -1,110 +1,75 @@
...
+|1
+|===
+
+Total possible score: 7 (100%)
```
Comment (amilan17) [inline KPI/B.adoc:54]: total score is 8

**Review by amilan17 — COMMENTED** [review]
small minor edits. see comments inline.

**Review by josusky — COMMENTED** [review]
The check "title exists" with score "0" is confusing. Probably the score "1" should have been "1". But as we have check if title has at least 3 words we could merge those two, could't we?

**Comment by tomkralidis** [comment]
Thanks @amilan17 / @josusky.  KPI B updated per your suggestions for review/comment.

**Review by amilan17 — APPROVED** [review]

**Review by josusky — APPROVED** [review]

**Pull request merged**